### DISCUSSION TITLE
Version 8.7.2 not fully published

### DISCUSSION THREAD

**Discussion by viceice** [ikvmnet/ikvm#451] [General]
The version 8.7.2 was not fully published and i'm having the issue in https://github.com/ikvmnet/ikvm/issues/440 so i can't upgrade to version 8.7+.

Are there any plans to fully pusblish version 8.7.2?

**Comment by martin-honnen** [comment]
I am also waiting for the full release, as far as I can tell from https://github.com/ikvmnet/ikvm/actions the build action so far has not run through completely but somehow failed.
Let's hope someone can fix that.

**Reply by wasabii** [reply]
Yeah. I've got three tests which magically started failing. On the main branch I restarted them a couple times and they worked eventually. So I thought, fine, I'll just release it, seems to be environment specific. Turns out the exact same code won't ever succeed now. I'm going to have to pull the release and fix em somehow.

But I've been very busy with paid work lately.

**Reply by viceice** [reply]
ok, thanks. I'm often very busy with paid work too. so nearly no time for open source 🤷‍♂️
